doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,113 @@
+
+| Factor | Notes       |
+|-------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Each [replica for a cluster](/sql/create-cluster/#replication-factor). | The credit usage for a cluster is measured at a one second granularity for each cluster replica. For a cluster replica, its credit usage begins when it is provisioned and ends when it is deprovisioned. |
+| [Indexes](/concepts/indexes/) and [materialized views](/concepts/views) | As data changes (insert/update/delete), [indexes](/concepts/indexes/) and [materialized views](/concepts/views) perform incremental updates to provide up-to-date results. That is: <br>• Indexes and materialized views perform work on "writes"<br>• Indexes and materialized views perform **no** work on "reads" from these objects; i.e., [`SELECT`s](/sql/select/) and [`SUBSCRIBE`s](/sql/subscribe/) from these objects are **free**. |
+| [Sources](/concepts/sources/) | Depends. |
+| [`SELECT`s](/sql/select/) and [`SUBSCRIBE`s](/sql/subscribe/)  |• [`SELECT`s](/sql/select/) and [`SUBSCRIBE`s](/sql/subscribe/) that do not use indexes and materialized views perform work. <br>• [`SELECT`s](/sql/select/) and [`SUBSCRIBE`s](/sql/subscribe/) that use indexes and materialized views are **free**.|
+| [Sinks](/concepts/sinks/) | Only small CPU/memory costs.|
+
+### Compute cost reductions
+
+Some considerations for lowering costs:
+
+- [Optimize queries](/transform-data/optimization/).
+
+- Avoid creating unnecessary indexes. For example,
+
+  - If you have created various views to act as the building blocks for a view
+    from which you will serve the results, indexing the serving view only may
+    be sufficient.
+
+  - In Materialize, underlying data structure that is being maintained for an
+    index (or a materialized view) can be reused by other views/queries. As
+    such, an existing index may be sufficient to support your queries.
+
+- For <redb>development</redb> clusters, you can set your replicas to zero when
+  not using your <redb>development</redb> clusters.
+
+## Additional references
+
+- https://materialize.com/pricing/
+
+- [How Materialized can lower the cost of freshness for data teams](https://materialize.com/promotions/cost-of-freshness/?utm_campaign=General&utm_source=documentation)
```

Comment:
Not sure how I feel about pointing to gated content from the documentation. We should at least make sure the gist of that content is captured in this page.